[E2E] APM Tests are failing

Could be related to #2298 ?

```
[2019-12-19T08:52:32.071Z] {"Time":"2019-12-19T08:52:30.599184079Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/ApmServer_version_should_be_the_expected_one","Output":"    --- PASS: TestCrossNSAssociation/ApmServer_version_should_be_the_expected_one (0.00s)\n"}
[2019-12-19T08:52:32.071Z] {"Time":"2019-12-19T08:52:30.599195506Z","Action":"pass","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/ApmServer_version_should_be_the_expected_one","Elapsed":0}
[2019-12-19T08:52:32.071Z] {"Time":"2019-12-19T08:52:30.599203048Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_accepted","Output":"    --- PASS: TestCrossNSAssociation/Events_should_be_accepted (0.00s)\n"}
[2019-12-19T08:52:32.071Z] {"Time":"2019-12-19T08:52:30.599212731Z","Action":"pass","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_accepted","Elapsed":0}
[2019-12-19T08:52:32.071Z] {"Time":"2019-12-19T08:52:30.599227177Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_eventually_show_up_in_Elasticsearch","Output":"    --- PASS: TestCrossNSAssociation/Events_should_eventually_show_up_in_Elasticsearch (9.20s)\n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599244039Z","Action":"pass","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_eventually_show_up_in_Elasticsearch","Elapsed":9.2}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599256559Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Output":"    --- FAIL: TestCrossNSAssociation/Events_should_be_forbidden (0.00s)\n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599264377Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Output":"        checks_apm.go:156: \n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599272002Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Output":"            \tError Trace:\tchecks_apm.go:156\n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599279612Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Output":"            \tError:      \tReceived unexpected error:\n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599286985Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Output":"            \t            \tinvalid character 'e' looking for beginning of value\n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599295807Z","Action":"output","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Output":"            \tTest:       \tTestCrossNSAssociation/Events_should_be_forbidden\n"}
[2019-12-19T08:52:32.072Z] {"Time":"2019-12-19T08:52:30.599326311Z","Action":"fail","Package":"github.com/elastic/cloud-on-k8s/test/e2e/apm","Test":"TestCrossNSAssociation/Events_should_be_forbidden","Elapsed":0}
```


7.1.1: https://devops-ci.elastic.co/blue/organizations/jenkins/cloud-on-k8s-stack/detail/cloud-on-k8s-stack/129/pipeline/37

--- a/test/e2e/test/apmserver/http_client.go
+++ b/test/e2e/test/apmserver/http_client.go
@@ -83,6 +83,9 @@ func (c *ApmClient) doRequest(context context.Context, request *http.Request) (*
 	// inject the authorization (secret token)
 	request.Header.Set("Authorization", c.authorizationHeaderValue)
 
+	// allow json content
+	request.Header.Set("Accept", "application/json")
+
 	return c.client.Do(withContext)
 }
 

